h/static/styles/annotation.scss

```diff
@@ -60,39 +90,61 @@ $annotation-card-left-padding: 10px;
   margin-left: 5px;
 }
 
+.annotation-body {
+  @include font-normal;
+  color: $grey-6;
+  // Align the baseline of the text with the bottom of a containing block element.
+  margin-top: $layout-h-margin - 5px;
+  margin-left: 0px;
+  margin-bottom: $layout-h-margin;
+  margin-right: 0px;
+}
+
 // the footer at the bottom of an annotation displaying
 // the annotation actions and reply counts
 .annotation-footer {
-  margin-bottom: 0;
+  @include font-normal;
+  color: $grey-5;
+  // Align the baseline of the text with the bottom of a containing block element.
+  margin-bottom: $layout-h-margin - 13px;
+  margin-top: $layout-h-margin;
 }
 
 .u-flex-spacer {
   flex-grow: 1;
 }
 
 .annotation-quote-list {
-  margin-top: 14px;
-  margin-bottom: 14px;
+  // Align the baseline of the text with the bottom of a containing block element.
```

Comment:
Same for this comment - I think it would be better for this comment to state the spacing that we are trying to achieve.